Dockerfile.prod

```diff
@@ -16,6 +16,6 @@ FROM node:14.17-alpine
 USER node
 COPY --chown=node:node --from=build /app /app
 WORKDIR /app
-ENV NODE_ENV development
+ENV NODE_ENV production
```

Comment:
Not sure why it was set to "development", would be good to know

Reply:
I think this was missed when the Dockerfiles were broken out. It was originally `production` in the micros Dockerfile branch but `development` in the TS Dockerfile branch.

Reply:
it was probably my bad when I moved docker files around, creating this .prod version.